doctoshotgun.py

```diff
@@ -196,6 +197,10 @@ def get_patients(self):
         self.master_patient.go()
 
         return self.page.get_patients()
+    
+    def remove_accents(self, string):
+        nfkd = unicodedata.normalize('NFKD', string)
+        return u"".join([c for c in nfkd if not unicodedata.combining(c)])
```

Comment:
Something like that:

```suggestion
    def normalize(string):
        nfkd = unicodedata.normalize('NFKD', string)
        normalized = u"".join([c for c in nfkd if not unicodedata.combining(c)])
        normalized = re.sub(r'\W', '-', normalized)
        return normalized.lower()
```